### ISSUE TITLE
[BUG] Cannot get card to show at all

### ISSUE THREAD

**Issue by lyricnz** [flixlix/energy-flow-card-plus#42]
**Describe the bug**
Just installed the card, attempting to get it to display on a dashboard.  Have removed all config to get minimal, but still doesn't work.

Config
```yaml
type: custom:energy-flow-card-plus
entities:
  grid:
    name: x
    entity:
      consumption: sensor.import_wh
energy_date_selection: false
title: energy-flow-card-plus
```

**To Reproduce**
Added card, configured just the grid import (Wh), everything else default.  When that didn't work, even removed all the other yaml, didn't help.

**Expected behavior**
A picture like the power-flow-card-plus (which is on the same dashboard)

**Screenshots**

The sensor data
<img width="911" alt="Screenshot 2023-06-08 at 12 44 29 pm" src="https://github.com/flixlix/energy-flow-card-plus/assets/122371/c13d9fb9-baa4-4287-a5cd-5507d4da6418">

Attempting to edit (no picture)
<img width="467" alt="Screenshot 2023-06-08 at 12 48 36 pm" src="https://github.com/flixlix/energy-flow-card-plus/assets/122371/5750d60e-1c0b-4cac-bb8a-09d7fa359e17">

**Desktop (please complete the following information):**
 - OS: macos
 - Browser: chrome
 - Version: Version 114.0.5735.106 (Official Build) (x86_64)

**Additional context**
Found some logs in chrome console
```
energy-flow-card-plus.js?hacstag=6382302440024:1 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'entity')
    at Qt.render (energy-flow-card-plus.js?hacstag=6382302440024:1:60458)
    at Qt.update (energy-flow-card-plus.js?hacstag=6382302440024:1:15132)
    at Qt.performUpdate (energy-flow-card-plus.js?hacstag=6382302440024:1:6493)
    at Qt.scheduleUpdate (energy-flow-card-plus.js?hacstag=6382302440024:1:6140)
    at Qt._$Ej (energy-flow-card-plus.js?hacstag=6382302440024:1:6048)
```

HACS report version is v0.0.2.4

**Please make sure all of this applies and check the checkboxes, by writing "X" inside of them.**

- [X] I understand that this is a channel for reporting bugs, not a support forum (If you need help configuring the card, go to: [Discussions](https://github.com/flixlix/energy-flow-card-plus/discussions)).

- [x] I have the latest version of the card installed.

- [x] I have cleared my browser cache **after** installing, updating and configuring the card.

**Comment by lyricnz** [comment]
FWIW even the minimal config from the README doesn't work.

**Comment by lyricnz** [comment]
TLDR I think the `energy_date_selection: false` doesn't work.  I manually edited the dashboard config, and added `type: energy-date-selection` and it started working.

I see in recent commit a disclaimer was added about this:
https://github.com/flixlix/energy-flow-card-plus/commit/d237d6b5276a1a8ed21b59f612dab59a90604b85

Which "energy integration" needs to be setup?  (my regular dashboard works fine, as does power-flow-card-plus)

**Comment by lyricnz** [comment]
With the date-picker manually added, it looks like it's working.

<img width="526" alt="Screenshot 2023-06-08 at 1 41 34 pm" src="https://github.com/flixlix/energy-flow-card-plus/assets/122371/038051ef-dccc-4aa3-a22a-1875496747b2">

**Comment by flixlix** [comment]
I think I'll add this disclaimer in the readme, to add the energy date picker

**Comment by lyricnz** [comment]
Yes, it's not a recommendation, it's a hard requirement
https://github.com/flixlix/energy-flow-card-plus#recommendation

**Comment by flixlix** [comment]
It's not a requirement, since you can configure this:

```yaml
energy_date_selection: false
```

In that case, you won't need any date picker

**Comment by lyricnz** [comment]
That's my point (the original defect). This didn't work.

**Comment by lyricnz** [comment]
This is probably the same as https://github.com/flixlix/energy-flow-card-plus/issues/38

**Comment by flixlix** [comment]
Which version of the card are you using?
I tried this exact config and just changed the entity id to something on my system and everything works like expected

**Comment by flixlix** [comment]
![Bildschirmfoto 2023-06-10 um 12 48 25](https://github.com/flixlix/energy-flow-card-plus/assets/61006057/bf79c85a-4738-4c21-a6cf-3296e5f978c4)
